ExtendedSplashScreen.sln

```diff
@@ -445,6 +445,8 @@ Global
 		{517BF627-6B4C-4488-B044-9FB92F2DE8A7}.AppStore|x86.Deploy.0 = Release|x86
 		{517BF627-6B4C-4488-B044-9FB92F2DE8A7}.Debug|Android.ActiveCfg = Debug|x86
 		{517BF627-6B4C-4488-B044-9FB92F2DE8A7}.Debug|Any CPU.ActiveCfg = Debug|x86
+		{517BF627-6B4C-4488-B044-9FB92F2DE8A7}.Debug|Any CPU.Build.0 = Debug|x86
```

Comment:
Change needed ?

Reply:
It was because by default, the UWP app could not be deployed. This is more handy!